lib/Makefile

```diff
@@ -7,6 +7,12 @@ else
 	OS_NAME=$(shell uname | tr '[:upper:]' '[:lower:]')
 endif
 
+ifeq ($(OS_NAME),darwin)
```

Comment:
This is an opportunity to also support `.dll`s for Windows.

Reply:
Sure, I can do it on another PR, but I'll need help of someone with a Windows machine to test it.